lib/stripe/converter.ex

```diff
@@ -0,0 +1,52 @@
+defmodule Stripe.Converter do
+
+  @doc """
+  Takes the module (e.g. `Stripe.Card`) and the response from Stripe and
+  returns a struct (e.g. `%Stripe.Card{}`) containing the Stripe response.
+  """
+  @spec stripe_map_to_struct(module, %{String.t => any}) :: struct
+  def stripe_map_to_struct(module, response) do
+    [_|struct_keys] = Map.keys(module.__struct__)
+
+    processed_map =
+      Enum.reduce(struct_keys, %{}, fn key, acc ->
+        value =
+          fetch_value(response, key)
+          |> convert_value()
+
+        value = Map.get(module.relationships, key)
```

Comment:
Above we newline on `value` but here we don't. Thoughts on newlining here as well for consistency?

Also, what are your thoughts on piping everything?
```
value = 
  module.relationships
  |> Map.get(key)
  |> build_struct(value)
```